tests/cases/standalone/common/alter/alter_table.result

```diff
@@ -215,7 +215,7 @@ SELECT * FROM grpc_latencies;
 | 2024-07-11T20:00:06 | host1 | GetUser     | 103.0   |
 +---------------------+-------+-------------+---------+
 
-ALTER TABLE grpc_latencies SET ttl = '10d';
+ALTER TABLE grpc_latencies SET ttl = '10000d';
```

Comment:
Why change this test, how does it relate to this PR?

Reply:
The TTL is too short, so the data expires and may be purged during sqlness test, and give an empty query result.